pages/api/utils/policy-utils.js

```diff
@@ -20,6 +23,9 @@ export const mapToClientModel = (policyResponse) => {
     name: policyResponse.name,
     description: policyResponse.description,
     regoContent: policyResponse.policy.regoContent,
+    latestVersion: policyResponse.currentVersion,
```

Comment:
Maybe a nitpick, but I think this should use the same term (current) instead of latest.